platform_services/index.js

```diff
@@ -320,7 +332,8 @@ module.exports.handler = (event, context, cb) => {
                 },
                 // Check if a service with same domain and service_name combination exists
                 validateServiceExists: function(onComplete) {
-                    crud.getList({ 'service': service_data.service, 'domain': service_data.domain }, function onServiceGet(error, data) {
+                    getAllRecords = getAllCheck(event.method);
+                    crud.getList({ 'service': service_data.service, 'domain': service_data.domain }, getAllRecords, function onServiceGet(error, data) {
```

Comment:
Here it should say: crud.getList({ 'service': service_data.service, 'domain': service_data.domain }, getAllRecords: true,